src/python/pants/backend/codegen/soap/java/register.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.backend.codegen.soap.java import rules as wsdl_java
+from pants.backend.codegen.soap.rules import rules as wsdl_rules
+from pants.backend.codegen.soap.target_types import WsdlSourcesGeneratorTarget, WsdlSourceTarget
+
+
```

Comment:
I don't really see anything here that would necessarily need to be marked experimental, so I'm ok skipping straight to stable for this.

To actually ship this backend in the pants distribution though, you'll need to make changes similar to #14407.

Reply:
actually this was an oversight, I intended to put in the experimental side. I'll change it to follow the approach in the referenced PR